todo/dist/index.js

```diff
@@ -1,33 +1,25 @@
-"use strict";
-var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
-    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
-    return new (P || (P = Promise))(function (resolve, reject) {
-        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
-        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
-        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
-        step((generator = generator.apply(thisArg, _arguments || [])).next());
-    });
-};
-Object.defineProperty(exports, "__esModule", { value: true });
-const hapi_1 = require("@hapi/hapi");
-const init = () => __awaiter(void 0, void 0, void 0, function* () {
-    const server = new hapi_1.Server({
+/*
+import { Server, Request, ResponseToolkit } from "@hapi/hapi";
+const init = async () => {
+    const server: Server = new Server({
         port: 5000,
         host: 'localhost'
     });
     server.route({
         method: 'GET',
         path: '/',
-        handler: (request, h) => {
+        handler: (request: Request, h: ResponseToolkit) => {
             return 'Hello World!';
         }
     });
-    yield server.start();
+    await server.start();
     console.log(`Running on ${server.info.uri}`);
```

Comment:
We shouldnt be committing `dist`. Also, I'm concerned that I'm seeing TypeScript in this file?

Reply:
The typescript is in the index.js file because I have index.ts commented out.  When typescript gets translated into javaScript because it is commented out no translation can take place. I forgot to remove the index.ts file so I will do that now.

Reply:
The dist. files got added because I left off the `/` in the .gitignore

Reply:
Ah, i didnt know that, thanks!